python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -2656,6 +2656,511 @@ def echo(self, x):
+                continue
+
+            # Collect nodes
+            if in_nodes_section:
+                node_match = node_pattern.match(line)
+                if node_match:
+                    node_id = node_match.group(1)
+                    node_names.add(node_id)
+
+            # Collect edges
+            if in_edges_section:
+                edge_match = edge_pattern.match(line)
+                if edge_match:
+                    from_node, to_node = edge_match.groups()
+                    edge_pairs.add((from_node, to_node))
+
+        return node_names, edge_pairs
+
+    def test_visualize_ascii_basic(self, ray_start_regular):
+        """
+        Expect output:
+            Nodes Information:
+            0 [label="Task 0
+            InputNode"]
+            1 [label="Task 1
+            Actor: 4b669e...
+            Method: echo"]
+            2 [label="Task 2
+            MultiOutputNode"]
+
+            Edges Information:
+            0 -> 1 [label=SharedMemoryType]
+            1 -> 2 [label=SharedMemoryType]
+
+            Experimental Graph Built:
+            0:InputNode
+            |
+            1:Actor:echo
+            |
+            2:MultiOutputNode
+        """
+
+        @ray.remote
+        class Actor:
+            def echo(self, x):
+                return x
+
+        actor = Actor.remote()
+
+        with InputNode() as i:
+            dag = actor.echo.bind(i)
+
+        compiled_dag = dag.experimental_compile()
+
+        # Call the visualize method
+        ascii_visualization = compiled_dag.visualize_ascii()
+
+        node_names, edge_pairs = TestVisualizationAscii.parse_ascii_visualization(
+            ascii_visualization
+        )
+        print(node_names, edge_pairs)
+        expected_nodes = {"0", "1", "2"}
+        assert expected_nodes.issubset(
+            node_names
+        ), f"Expected nodes {expected_nodes} not found."
+
+        expected_edges = {("0", "1"), ("1", "2")}
+        assert expected_edges.issubset(
+            edge_pairs
+        ), f"Expected edges {expected_edges} not found."
+
+        compiled_dag.teardown()
+
+    def test_visualize_ascii_multi_return(self, ray_start_regular):
+        """
+        Expect output:
+            Nodes Information:
+            0 [label="Task 0
+            InputNode"]
+            1 [label="Task 1
+            Actor: 68719a...
+            Method: return_two"]
+            2 [label="Task 2
+            ClassMethodOutputNode[0]"]
+            3 [label="Task 3
+            ClassMethodOutputNode[1]"]
+            4 [label="Task 4
+            MultiOutputNode"]
+
+            Edges Information:
+            0 -> 1 [label=SharedMemoryType]
+            1 -> 2 [label=SharedMemoryType]
+            1 -> 3 [label=SharedMemoryType]
+            2 -> 4 [label=SharedMemoryType]
+            3 -> 4 [label=SharedMemoryType]
+
+            Experimental Graph Built:
+            0:InputNode
+            |
+            1:Actor:return_two
```

Comment:
How about the following format:
1:Actor_a0b3:return_two
a0b3 is the prefix of the actor ID